Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -194,4 +196,5 @@
 #
 reconstruction_standard_candle = cms.Sequence(localreco*globalreco*vertexreco*recoJetAssociations*btagging*electronSequence*photonSequence)
 
-
+from Configuration.StandardSequences.Eras import eras
+eras.phase2_common.toReplaceWith( localreco, _phase2_localreco )
```

Comment:
submitted as a separate commit so it can be reverted back to the Era-dependent version if necessary